torch/_export/passes/add_runtime_assertions_for_constraints_pass.py

```diff
@@ -139,6 +141,11 @@ def call(self, graph_module):
         # Add runtime asserts for inline constraints
         val = super().call(graph_module)
 
+        # Sometimes this pass would return a wrong graph where we have mismatched
+        # node names in signature. Before we fix it, let's just skip it.
```

Comment:
Do you have a repro for this?

Reply:
yeah! unfortunately it's in the internal, if you want I can show you the buck target.